decidim/decidim
# Fix usage of `private_non_transparent_space?`

#### :tophat: What? Why?
Back in #13169 the method `private_non_transparent_space?` has been added. I accidentally found out that sometimes, it raises an error when trying to create a meeting in initiatives or conferences spaces. 

An alternate fix for this bug would be to implement the `private_space?` method in both models ( `Decidim::Initiative` and `Decidim::Conference`) that would return false, like: 

```ruby
def private_space?
  false
end 
```

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #13169

#### Testing
1. go to nightly's admin 
2. navigate to Conference / Components / Meetings / new 
3. See the page is raising an error 
4. Apply the patch
5. Refresh, and see there is no error. 

### :camera: Screenshots

![image](https://github.com/user-attachments/assets/09de2c2f-877a-4d37-a562-ff64e253e3be)

:hearts: Thank you!
